match/new_orleans_pd.py

```diff
@@ -5,9 +5,61 @@
     JaroWinklerSimilarity,
     DateSimilarity,
     ColumnsIndex,
+    Swap,
 )
 from lib.post import extract_events_from_post, load_for_agency
 import pandas as pd
+from lib.clean import canonicalize_officers
+
+
+def deduplicate_pprr(pprr):
+    df = (
+        pprr.loc[
+            pprr.uid.notna(),
+            ["employee_id", "first_name", "last_name", "middle_name", "uid"],
+        ]
+        .drop_duplicates()
+        .set_index("uid", drop=True)
+    )
+
+    matcher = ThresholdMatcher(
+        ColumnsIndex("employee_id"),
+        {
+            "first_name": JaroWinklerSimilarity(),
+            "last_name": JaroWinklerSimilarity(),
+        },
+        df,
+        variator=Swap("first_name", "last_name"),
+    )
```

Comment:
Are there people with the same `employee_id` but different names? I find this index choice rather odd.

Reply:
I haven't actually checked, but I know that it is very possible because of the timeframe for which we have data, therefore, I made this change to err on the side of caution. Would you prefer that I check first?

Reply:
No need to check first. I think it is fine if there's no instance where the same person has different employee ids.